src/AppInstallerCommonCore/Public/winget/ManifestLocalization.h

```diff
@@ -25,9 +25,17 @@ namespace AppInstaller::Manifest
         ShortDescription,
         Description,
         Tags,
+        Agreements,
         Max
     };
 
+    struct Agreement
+    {
+        std::optional<string_t> Label;

```

Comment:
>std::optional [](http://example.com/codeflow?start=8&length=13)

I understand using std::optional is more accurate to differentiate empty vs. not exist, but everywhere else in the manifest uses empty string to denote field not exist, so maybe we keep the consistency for now